src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding-left: 0;
+  box-sizing: border-box;
+
+  list-style: none;
+  border-radius: $border-radius;
+  border: 1px solid #f3f3f3;
+
+  &-container {
+    display: flex;
+    flex-wrap: wrap;
+    width: 944px;
+    padding: 50px 40px;
+    margin: 0 auto;
+  }
+
```

Comment:
it's not `card` element or modifier. It's a separate block witch use `-` in name consisted from 2 words.
Move this code to a separate file.
Rewatch info about naming in [this video](https://mate.academy/learn/css-advanced/bem#/video/920)